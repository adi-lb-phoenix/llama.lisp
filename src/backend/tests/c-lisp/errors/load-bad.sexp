(c-lisp
    (define ((main void) (argc int) (argv (ptr (ptr int))))
        (declare a (ptr int))
        (load argv a)))
