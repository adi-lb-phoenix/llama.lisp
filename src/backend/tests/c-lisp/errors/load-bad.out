Error in statement: ['load', 'argv', 'a']
CodegenError: Bad load expression: ['load', 'argv', 'a']
